code/src/pages/MovieDetail.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable camelcase */
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+
+export const MovieDetail = () => {
+  const { id } = useParams()
+  const [info, setInfo] = useState([])
+
+  useEffect(() => {
+    const APIKey = 'c03059469ca2a2b2651ea90d6f8361ab'
+    const movieAPI = `https://api.themoviedb.org/3/movie/${id}?api_key=${APIKey}&language=en-US`
+
+    fetch(movieAPI)
+      .then((res) => res.json())
+      .then((json) => {
+        setInfo(json)
+      })
+  }, [id])
+
+  return (
+    <section
+      className="container"
+      key={id}
+      style={{
```

Comment:
If you see that you need to pass more than one style inside `style` attribute, then consider moving your code to separate variable, like this :

```
const myCustomStyles = {
        backgroundImage: `linear-gradient(rgba(0, 0, 0, 0) -70%, rgb(0, 0, 0) 100%), url(https://image.tmdb.org/t/p/w1280/${info.backdrop_path})`,
        minHeight: '100vh',
        backgroundSize: 'cover',
        backgroundRepeat: 'no-repeat'
}

return (
    <section style={myCustomStyles}>
        ...
    </section>
)
```

Reply:
done. moved styling into a variable.